disperser/dataapi/subgraph_client.go

```diff
@@ -97,6 +98,10 @@ type (
 		// The operatorIds of nonsigners for the batch.
 		NonSigners []string
 	}
+	Reservation struct {
```

Comment:
Should we also throw reservation symbols in here? Would allow us to monitor total allocated capacity.

Reply:
+1. How are the metrics meant to be used? I'm assuming this is purely for us internally to add grafana alerts when a reservation is about to expire?

Would we ever want to expose reservations info on the blob explorer? If so then we might as well read all the info exposed per reservation by the contract/subgraph?

Reply:
First use case is for reservation expiration tracking internally but we can extend the scope to any blob explorer requirements in future PRs.

Reply:
68084d22e72744a0e3e76d62ac3a92bbed965887